Objects/object.c

```diff
@@ -136,6 +136,21 @@ Py_DecRef(PyObject *o)
     Py_XDECREF(o);
 }
 
+#undef Py_NewRef
+#undef Py_XNewRef
```

Comment:
Perhaps redefine these 2 macros after defining the 2 functions, so this file can enjoy the `static inline` version as well?

Reply:
Good idea. I moved the definitions at the bottom of object.c, so object.c gets the fast static inlined flavor as well.

Reply:
> I moved the definitions at the bottom of object.c, 

Even better :)